crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -366,10 +367,19 @@ fn check_limits(
     pos: Option<Pos>,
     ctx: &ExtensionContext<'_>,
 ) -> ServerResult<()> {
+    let query_uuid = ctx
+        .data_opt::<QueryUuid>()
+        .map(|id| id.uuid.clone())
+        .unwrap_or_default();
```

Comment:
what is the default value? if we end up having a bunch of Nones for uuids, then we'd have a bunch of data mapped to this default value, which doesn't seem ideal